atom/common/options_switches.cc

```diff
@@ -193,6 +196,7 @@ const char kBackgroundColor[] = "background-color";
 const char kPreloadScript[] = "preload";
 const char kPreloadScripts[] = "preload-scripts";
 const char kNodeIntegration[] = "node-integration";
+const char kDisableRemoteModule[] = "disable-remote-module";
```

Comment:
This CLI flag is the last reference to "disable", any reason this can't be `enable-remote-module` with `=false` / `=true`

Reply:
I had issues when I've tried to use `--enable-remote-module`, seems like in some cases it would not be set for devtools or something and tests were failing.